.github/workflows/casual-build-beta.yaml

```diff
@@ -281,7 +281,7 @@ jobs:
           tag_prefix: ""
 
   publish-version:
-    runs-on: self-hosted
+    runs-on: [self-hosted, casual, publish]
```

Comment:
Is `publish` like a tag you've added to the original runner?

Reply:
Yes, Its any kind of label I have added to the runner "identification". And if you have an array of requested labels, all have to match the specific runner.